src/libaktualizr-posix/ipuptanesecondary.cc

```diff
@@ -163,37 +220,136 @@ data::ResultCode::Numeric IpUptaneSecondary::install(const std::string& target_n
   return static_cast<data::ResultCode::Numeric>(r->result);
 }
 
-Manifest IpUptaneSecondary::getManifest() const {
-  LOG_DEBUG << "Getting the manifest from secondary with serial " << getSerial();
+data::ResultCode::Numeric IpUptaneSecondary::install_v2(const Uptane::Target& target) {
+  data::ResultCode::Numeric data_delivery_result = data::ResultCode::Numeric::kDownloadFailed;
+
+  LOG_INFO << "Invoking an image delivery of the target  " << target.filename() << " to/on the Secondary "
```

Comment:
Maybe something like "Instructing Secondary <xyz> to receive target <abc>" would be clearer?

Also, don't these image receiving steps belong in the download phase, not installation? That might be out of scope here, but I'm trying to remember why we handle this here at all.

Reply:
ok